src/input_xml.F90

```diff
@@ -5824,23 +5797,19 @@ subroutine read_ce_cross_sections(libraries, library_dict)
 
           ! Determine if minimum/maximum energy for this nuclide is greater/less
           ! than the previous
-          energy_min_neutron = max(energy_min_neutron, nuclides(i_nuclide) % energy(1))
-          energy_max_neutron = min(energy_max_neutron, nuclides(i_nuclide) % energy(&
-               size(nuclides(i_nuclide) % energy)))
+          if (size(nuclides(i_nuclide) % grid) >= 1) then
```

Comment:
Yes, in fact. If you have a material in your materials.xml file that doesn't actually appear in the model and it has nuclides that no other material does, these nuclides will end up getting loaded, but because of the way we choose temperatures (looping over cells), there won't be any temperatures to load. Ergo, there are no energy grids.